linux-setup.sh

```diff
@@ -53,6 +53,20 @@ install_go() {
     fi
 }
 
+# Builds and installs `mkcert` which is used by the following things in
+# webapp:
+# - https://khanacademy.dev
+# - the "Vitejs Directly" option in the dev support bar
+install_mkcert() {
+    if ! which mkcert >/dev/null; then
+        git clone https://github.com/FiloSottile/mkcert && cd mkcert
```

Comment:
This will likely result in there being a `mkcert` dir added to the khan-dotfiles directory which is not good.  I'm going to follow the pattern we use to install watchman to create a directory in `builddir="$DEVTOOLS_DIR/mkcert/"` instead.